chore(styles, documentation): move focus-ring bootstrap helper class to DS (#4543)
swisspost/design-system: .changeset/late-days-approve.md

---

> '@swisspost/design-system-documentation': patch
> '@swisspost/design-system-components': patch
> '@swisspost/design-system-styles': patch
> ---
> 
> Moved the Bootstrap `.focus-ring` helper class to design system styles and documented it.

Reviewer: ```suggestion
'@swisspost/design-system-documentation': minor
'@swisspost/design-system-styles': patch
---

Updated the styles for `.focus-ring` and added documentation.
```
